docs/operations/hotplug_volumes.md

```diff
@@ -76,6 +76,16 @@ The format and length of serials are specified according to the libvirt document
 
 ```
 
+#### Supported Disk types
+Kubevirt supports hotplugging disk devices of [disk](https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#disk) and [lun](https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#lun) types. As with regular volumes, using `disk` will expose the hotplugged volume as a regular disk, while using `lun` allows additional functionalities like the execution of iSCSI commands.
```

Comment:
`KubeVirt supports hotplugging disk devices of type [disk](...) and [lun](...)`

`As with other volumes, using type 'disk' will expose ...`